src/NuGet.Server/Infrastructure/ServerPackage.cs

```diff
@@ -79,8 +39,10 @@ public ServerPackage(IPackage package, PackageDerivedData packageDerivedData)
             MinClientVersion = package.MinClientVersion;
             ReportAbuseUrl = package.ReportAbuseUrl;
             DownloadCount = package.DownloadCount;
-            IsAbsoluteLatestVersion = package.IsAbsoluteLatestVersion;
-            IsLatestVersion = package.IsLatestVersion;
+            SemVer1IsAbsoluteLatest = false;
+            SemVer1IsLatest = false;
+            SemVer2IsAbsoluteLatest = false;
+            SemVer2IsLatest = false;
```

Comment:
> If I remember right these flags existed because OData will sort the versions as strings. So to find the actual latest version using correct semver sort it was easier for OData to just filter on these bools.

Currently sort order is incorrect as it is lex order no SemVer 2.0.0 order but this is tracked by:
https://github.com/NuGet/NuGetGallery/issues/3438

The `IsLatestVersion` and `IsAbsoluteLatestVersion` filters depend on there being properties of the same name. Example client code:
https://github.com/NuGet/NuGet.Client/blob/6bd6cecdfedb8efcfa6b05c3694f05d6898006a1/src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/V2FeedParser.cs#L32

This is part of the contract and cannot change.

> Do these end up in the feed xml?

They end up in the feed (OData) XML in the form of `IsLatestVersion` and `IsAbsoluteLatestVersion` properties.